data/svgs/svg-lookup.ini

```diff
@@ -70,7 +70,7 @@ Svg=logitech-g703.svg
 DeviceMatch=usb:046d:c081
 Svg=logitech-g900.svg
 
-[Logitech G900, G903, G603, G403]
+[Logitech Device (via Lightspeed connector)]
 DeviceMatch=usb:046d:c539
 Svg=logitech-g900.svg
```

Comment:
The `Svg` entry still points to the `logitech-g900.svg` file and the new one isn't used anywhere afaict from this diff.

It helps if you forget anything about what libratbag does and focus only on piper. The only identifier we get from ratbagd is the `usb:1235:abcd` string. Now we need to figure out which SVG to render for that string. And that's the `Svg=` entry. The .ini file we have here uses the section headers so that the puny humans can figure out what's going on (instead of say a comment). The rest is just a lookup between the match string and whatever svg we want to render. Changing the section header is like changing a comment but has no functional implications.